### PR TITLE
extract context-menu related code into a separate package

### PR DESCRIPTION
**Pull request by parro-it**
* deleted context-menu.js file
* removed activation code from index
* added a note to readme advising feature removal

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

File: readme.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Force reload the window.
 
 Right-click on an HTML element when DevTools is open to inspect it.
 
+Note: this features has been moved to [its own package](https://github.com/parro-it/debug-menu). Please refer to it for related issues.
```
Comment (sindresorhus) [inline readme.md]: Can you just remove this "Inspect element" section and add your module at the bottom in a `Related` section. Like, eg https://github.com/sindresorhus/got#related

Comment (parro-it) [inline readme.md]: Ah yes, I always find `related` section in your repos useful!

**Comment by parro-it** [comment]
Should be mergeable now...

File: readme.md [hunk 2]
```diff
@@ -41,16 +37,16 @@ $ npm install --save electron-debug
...
 
 app.on('ready', function () {
-	win = new BrowserWindow({
-		width: 800,
-		height: 600
-	});
+  win = new BrowserWindow({
+    width: 800,
+    height: 600
+  });
```
Comment (sindresorhus) [inline readme.md]: Don't change unrelated things.

File: readme.md [hunk 3]
```diff
@@ -60,6 +56,13 @@ app.on('ready', function () {
...
+## Related
+
+- [debug-menu](https://github.com/parro-it/debug-menu) - Chromium-like debug context-menu for [Electron](http://electron.atom.io)
+
+
 ## License
 
 MIT © [Sindre Sorhus](http://sindresorhus.com)
+
+
```
Comment (sindresorhus) [inline readme.md]: Don't change unrelated things.

File: readme.md (hunk 2 above)
Comment (parro-it) [inline readme.md]: I usually try not to, but sublime text get in the way with tap indentation...

Comment (sindresorhus) [inline readme.md]: http://editorconfig.org/ ;)

Comment (parro-it) [inline readme.md]: I already use it, but I don't know why, on my linux PC sublime give precedence to my user preferences. On my MAC in office, it correctly does the opposite...

**Pull request closed without merging**